evaluations/pose_graph_synthetic.sh

```diff
@@ -6,11 +6,14 @@
 #  solver_device: {cuda, cpu}
 #  batch_size: {8, 16, 32, 64, 128, 256}
 #  num_poses: {128, 256, 512, 1024, 2048, 4096}
-
+#
+#  When solver_device=cuda, solver_type can be lucuda|baspacho
+#  otherwise, it's ignored
 python examples/pose_graph/pose_graph_synthetic.py \
        inner_optim.solver=sparse \
        loop_closure_ratio=0.2 \
        solver_device=cuda \
-       dataset_size=256 \
-       batch_size=128 \
-       num_poses=256
+       dataset_size=32 \
+       batch_size=32 \
+       num_poses=4096 \
+       solver_type=baspacho
```

Comment:
Like @luisenp mentioned we should swap to a more reasonable default that will always work out of the box?

Reply:
oh sounds good.